### PR TITLE
removes \n from changelog content

### PR DESCRIPTION
**Pull request by RishabhTayal**
Currently the changelog content contains a new line. This is causing `pilot` to fail when we set the `release_notes` option. This should fix it.

### CODE REVIEW

**Comment by pajapro** [comment]
Hey @RishabhTayal,

Thanks a lot for your PR. I have two concerns about this addition:

1. the `\n` is added on purpose in order to visually separate sections by a new line
2. unit tests [keep failing](https://travis-ci.org/pajapro/fastlane-plugin-changelog/builds/296507309?utm_source=github_status&utm_medium=notification)

Do you think it would make sense to encapsulate this addition into a parameter (e.g.: `include_newline_separator` of `Boolean` value)? This way, it wouldn't break other users setup, while you could still leverage from it.

**Comment by RishabhTayal** [comment]
This only removes the \m from the content that's returned. Not from sections. 

The test cases are failing even for master. I fixed the test cases broken because of my changes. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Fixed:
+- removes `\n` from changelog content
```
Comment (pajapro) [inline CHANGELOG.md]: Would this be more expressive:

> Remove `\n` (newline) from `read_changelog` action's result.

Comment (RishabhTayal) [reply CHANGELOG.md]: This sounds better.

**Comment by RishabhTayal** [comment]
Thanks for merging! 🎉

**Pull request merged**